index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta charset="UTF-8">
+        <title>Hello Microverse</title>
+    </head>
+    <body>
+        <main>
+            <header>
+                
+            </header>
+        </main>
```

Comment:
- [OPTIONAL] According to the project requirements, Your index.html file should include a header with the text "Hello Microverse!". and you were asked to use an `h1` tag for that. Kindly add this 👨🏻‍🎨 

```
        <header>
            <h1> Hello Microverse! </1>
        </header>
```

### This is only optional because due to oversight the previous reviewer probably missed them. However, it is important.